src/localization/locales/en-US.json

```diff
@@ -422,7 +422,28 @@
         "map_create_document_title": "Create Map: {{name}}",
         "map_view_document_title": "View Map: {{name}}",
         "fetchVisualizationConfig.not_found": "Map not found",
-        "addVisualizationConfig.unique": "The name “{{body.variables.input.title}}“ is in use by another map. Please use a different name."
+        "addVisualizationConfig.unique": "The name “{{body.variables.input.title}}“ is in use by another map. Please use a different name.",
+        "tab_files_zero": "Upload files",
+        "tab_files": "Upload files ({{count}})",
+        "tab_links_zero": "Share links",
+        "tab_links": "Share links ({{count}})",
+        "upload_summary_files_one": "1 file",
+        "upload_summary_files": "{{count}} files",
+        "upload_summary_links_one": "1 link",
+        "upload_summary_links": "{{count}} links",
+        "upload_summary_separator": " and ",
+        "upload_summary_success_zero": "",
+        "upload_summary_success_one": "{{successCounts}} has been shared sucessfully. ",
+        "upload_summary_success": "{{successCounts}} have been shared sucessfully. ",
+        "upload_summary_error_zero": "",
+        "upload_summary_error": "{{errorCounts}} could not be shared. ",
+        "upload_summary_zero": "",
+        "upload_summary": "[TBD] $t(sharedData.upload_summary_success, {\"count\": {{successCount}} })$t(sharedData.upload_summary_error, {\"count\": {{errorCount}} })Fix the issues and share again or skip sharing files or links with issues.",
```

Comment:
What is supposed to happen here?

Reply:
@paulschreiber This is the summary message that appears above the upload button
<img width="717" alt="image" src="https://user-images.githubusercontent.com/1316782/204875326-814860e3-d10e-486b-abec-4b7c5b132909.png">